data_model/src/expression.rs

```diff
@@ -400,6 +399,28 @@ macro_rules! expression_serde_internal_repr {
                     }
                 }
             }
+
+            pub mod reference {
+                use super::*;
+
+                #[derive(::serde::Serialize)]
+                pub enum $i <'re> {
+                    $(
+                        $(#[$var_meta])*
```

Comment:
Field attributes without enum attributes could lead to problems.

Reply:
Indeed. Then I brign back meta for the enum

Reply:
Or better remove field attributes, because they cause some problems with lifetimes